featuretools/tests/primitive_tests/test_groupby_transform_primitives.py

```diff
@@ -352,6 +355,31 @@ def get_function(self):
         assert ((pd.isnull(x) and pd.isnull(y)) or x == y)
 
 
+def test_groupby_multi_output_stacking(es):
+    def test_time(x):
+        return x
+
+    TestTime = make_trans_primitive(
+        function=test_time,
+        input_types=[Datetime],
+        return_type=Numeric,
+        number_output_features=6,
+    )
+
+    fl = dfs(
+        entityset=es,
+        target_entity="sessions",
+        agg_primitives=[],
+        trans_primitives=[TestTime, CumSum],
```

Comment:
`CumSum` should be in the `groupby_trans_primitives` list instead of `trans_primitives` so that we are testing a GroupByTransform feature instead of a Transform feature